lib/logflare/backends/adaptor/postgres_adaptor/log_event.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Logflare.Backends.Adaptor.PostgresAdaptor.LogEvent do
+  use Ecto.Schema
+  import Ecto.Changeset
+  @primary_key {:id, :string, []}
+  schema "log_events" do
```

Comment:
the [dynamic table PR](https://github.com/Logflare/logflare/pull/1580) will remove this right?

Reply:
Yes 👍 and it will be the next one to be merged since it impacts everything